src/components/notifications/notifications.svc.coffee

```diff
@@ -0,0 +1,36 @@
+angular.module 'mnoUiElements'
+  .service 'Notifications', ($log, toastr, MnoeNotifications) ->
+
+    NOTIFICATION_TYPE_MAPPING = {
+      reminder: 'info',
+      due_date: 'warning',
+      status_change: 'info',
+    }
+
+    @init = () ->
+      $log.debug("Notifications are enabled")
+      MnoeNotifications.get().then(
+        (response)->
+          notifications = response.data.plain()
+          for notification in notifications
+            notification_type = notification.notification_type
+            method = NOTIFICATION_TYPE_MAPPING[notification_type]
+            message = notification.message.split("\n").join("</br>")
+            title = notification.title
+            onHidden = ->
+              params = {object_id: notification.object_id, object_type: notification.object_type, notification_type: notification_type}
+              MnoeNotifications.notified(params)
+            toastr[method](message,title, {
+              closeButton: true,
+              autoDismiss: false,
+              timeOut: null,
+              tapToDismiss: true,
+              extendedTimeOut: 1000000,
```

Comment:
You can disable the timeouts using `0` (cf. https://github.com/CodeSeven/toastr#timeouts)
I tried this in the console and it is working perfectly:
```
toastr.info('<input type="checkbox" checked> Success!', 'With HTML', {
  allowHtml: true,
  extendedTimeOut: 0,
  timeOut: 0,
  closeButton: true
});
```

Please fix this to avoid a wrong value (`null`) and a magic number (`0`).